misk-metrics-digester/src/test/kotlin/misk/metrics/digester/RegistryTest.kt

```diff
@@ -0,0 +1,32 @@
+package misk.metrics.digester
```

Comment:
Increase the test coverage to at least have parity with the Go implementation.

Reply:
Ping. Please take a look at the test cases covered in the Go implementation. There are edge cases that are not tested here, and the implementation needs some fixes for them.